mne/viz/_brain/_timeviewer.py

```diff
@@ -744,6 +753,7 @@ def help(self):
             ('i', 'Toggle interface'),
             ('s', 'Apply auto-scaling'),
             ('r', 'Restore original clim'),
+            ('c', 'Clear all traces'),
```

Comment:
pressing c is not working for me. Did you forget to configure the callback?

Reply:
also pressing i or s does not work anymore for me :(

Reply:
Did you try with pyvista 0.23.1?

Reply:
This is now the recommended version:
https://github.com/mne-tools/mne-python/issues/7162#issuecomment-586287143